angular-mentoring/src/app/header/i-user.ts

```diff
@@ -0,0 +1,5 @@
+export interface IUser {
```

Comment:
In most cases, it doesn't make sense to put interfaces in the separate file. Consider moving it to the class